### PR TITLE
merge "jdk8u272-ga" from openjdk

### PR DESCRIPTION
**Pull request by ziyilin**

### CODE REVIEW

**Comment by joeyleeeeeee97** [comment]
Tests verified OK.

**Review by D-D-H — COMMENTED** [review]
jfrChunkState.[hc]pp should be removed.

**Comment by ziyilin** [comment]
> jfrChunkState.[hc]pp should be removed.

removed

**Review by D-D-H — APPROVED** [review]

File: make/aix/makefiles/vm.make [hunk 1]
```diff
@@ -147,7 +147,11 @@ CFLAGS += -DALLOW_OPERATOR_NEW_USAGE
 LIBJVM_DEBUGINFO   = lib$(JVM).debuginfo
 LIBJVM_DIZ         = lib$(JVM).diz
 
+<<<<<<< HEAD
```
Comment (kuaiwei) [inline make/aix/makefiles/vm.make:150]: It should be removed.

File: make/aix/makefiles/vm.make [hunk 2]
```diff
@@ -147,7 +147,11 @@ CFLAGS += -DALLOW_OPERATOR_NEW_USAGE
...
 ifeq ($(ENABLE_JFR),false)
+=======
```
Comment (kuaiwei) [inline make/aix/makefiles/vm.make:152]: Remove these line

**Pull request merged**